.travis.yml

```diff
@@ -6,40 +6,32 @@ branches:
     - develop
     - master
     - /^(?i:release|hotfix).*$/
-services:
-  - docker
 env:
   global:
     - GIT_SHA=$( git rev-parse HEAD )
-    - IT_CLIENT_HOST=client
     - BRANCH_NAME=${TRAVIS_PULL_REQUEST_BRANCH:=$TRAVIS_BRANCH}
-before_install:
-  - mkdir -p integration-tests
-  - mv !(integration-tests|node_modules) integration-tests
-  # fetch switch_branch script
-  - curl https://gist.githubusercontent.com/schul-cloud-bot/26e57e99fdae76f30927089c9dd99df7/raw/a590139da5f969a6d330f38ab2d85d3769c86f3f/switch_branch.sh > switch_branch.sh
-  - chmod 700 switch_branch.sh
-  # clone other required repositories and try to switch to branch with same name as current one
-  - git clone https://github.com/schul-cloud/docker-compose.git docker-compose
-  - sh switch_branch.sh "docker-compose" "$BRANCH_NAME"
-  - git clone https://github.com/schul-cloud/schulcloud-client.git schulcloud-client
-  - sh switch_branch.sh "schulcloud-client" "$BRANCH_NAME"
-  - git clone https://github.com/schul-cloud/schulcloud-server.git schulcloud-server
-  - sh switch_branch.sh "schulcloud-server" "$BRANCH_NAME"
-install:
-  # TODO use "Images" Dockerfile to improve speed
-  # - cd docker-compose && docker-compose -f docker-compose.integration-test-Images.yml up -d && cd ..
-  - docker-compose -f docker-compose/docker-compose.integration-test-Build.yml up -d
-  - cd integration-tests && npm ci && cd ..
-script:
-  - echo $TRAVIS_BRANCH
-  - echo $TRAVIS_TAG
-  - cd integration-tests
-  - npm run test
-  - cd ..
-after_script:
-  - npm i -g @adrianjost/report-viewer
-  - rv-upload -F **/reports/**/*.html
+jobs:
+  include:
+    - stage: run parallel
+      name: test:integration
+      script:
+        - echo $TRAVIS_BRANCH
+        - echo $TRAVIS_TAG
+        - curl https://gist.githubusercontent.com/schul-cloud-bot/a849c38804174f99ca7818782bf03c00/raw/index.sh > integration-test.sh
```

Comment:
would like to use a deployment project instead where this file is stored, same for depploy.sh-scripts we have in other places to have this decoupled from the projects itself. we could replace the url just by an env to modify it later but for now it's fine :)

Reply:
The env variable is actually a great Idea! I will write a ticket for that